src/ray/core_worker/core_worker.cc

```diff
@@ -308,6 +308,8 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
       task_execution_service_work_(task_execution_service_),
       resource_ids_(new ResourceMappingType()),
       grpc_service_(io_service_, *this) {
+  RAY_LOG(INFO) << "Constructing CoreWorker, worker_id: " << worker_id;
```

Comment:
Can we actually add a pid here too?

Reply:
The pid is already logged in `CoreWorkerProcess::CoreWorkerProcess`.